src/ert/scheduler/__init__.py

```diff
@@ -20,53 +20,48 @@ def create_driver(config: QueueConfig) -> Driver:
     if config.queue_system == QueueSystem.LOCAL:
         return LocalDriver()
     elif config.queue_system == QueueSystem.TORQUE:
-        queue_config = {
-            key: value
-            for key, value in config.queue_options.get(QueueSystem.TORQUE, [])
-        }
-        num_nodes: Optional[str] = queue_config.get("NUM_NODES")
-        num_cpus_per_node: Optional[str] = queue_config.get("NUM_CPUS_PER_NODE")
+        queue_options = config.queue_options_as_dict
```

Comment:
Looks fine, but why do we want this specific change?

Reply:
Just for better understanding and readability of the code. Currently we are creating dictionaries called `queue_config` and they only contain queue_options key-value pairs. I figured, maybe I was wrong here that it would be more understandable to call it queue_options given that is what they represent.

Reply:
ah, I think I misread the diff..